e2e/Microsoft.Azure.Devices.E2ETests/MessageE2ETests.cs

```diff
@@ -566,6 +565,42 @@ public static void ClassCleanup()
                 TestUtil.DefaultDelayInSec);
         }
 
+        [TestMethod]
+        [TestCategory("Message-E2E")]
+        [TestCategory("Timeout")]
+        [ExpectedException(typeof(TimeoutException))]
+        public async Task Message_TimeOutReachedResponse()
+        {
+            await FastTimeout();
+        }
+
+        [TestMethod]
+        [TestCategory("Message-E2E")]
+        [TestCategory("Timeout")]
+        // Should not have any exceptions thrown. 
```

Comment:
@ryan-z-johnson, shouldn't this timeout after [100 seconds](https://github.com/Azure/azure-iot-sdk-csharp/blob/9272857f1065585f2c4804dd5836b0ea95584ee9/service/Microsoft.Azure.Devices/AmqpServiceClient.cs#L20)?

Reply:
Or is it because the server responds well within that time?

Reply:
Responds well within that time.